config/share/settings-env/input_headers.json

```diff
@@ -26,7 +26,8 @@
     "Instill-User-Id",
     "Instill-Visitor-Uid",
     "Instill-Return-Traces",
-    "Instill-Share-Code"
+    "Instill-Share-Code",
+    "X-Use-SSE"
```

Comment:
We currently use the `Instill-` prefix for our custom headers and should maintain this convention.

Reply:
Hi @tillknuesting ,
I noticed that you changed the header to `Instill-Use-SSE` in the pipeline-backend, so we need to update it here accordingly.